internal/opa/rule/rule.go

```diff
@@ -39,6 +39,13 @@ func description(a *ast.AnnotationsRef) string {
 	return a.Annotations.Description
 }
 
+func effectiveOn(a *ast.AnnotationsRef) string {
+	if value, ok := a.Annotations.Custom["effective_on"].(string); ok {
```

Comment:
This needs something like, to prevent the nil reference error:
```suggestion
	if a == nil {
		return ""
	}
	if value, ok := a.Annotations.Custom["effective_on"].(string); ok {
```

Reply:
Thanks! I'll try that - the `shortName` function has a similar condition.